PowerShell/ScubaGear/baselines/Discontinued Policies.md

```diff
@@ -0,0 +1,78 @@
+**`TLP:CLEAR`**
+# Discontinued CISA M365 Security Configuration Baseline Policies
+
+The Secure Cloud Business Applications (SCuBA) project run by the Cybersecurity and Infrastructure Security Agency (CISA) provides guidance and capabilities to secure federal civilian executive branch (FCEB) agencies' cloud business application environments and protect federal information that is created, accessed, shared, and stored in those environments.
+
+The CISA SCuBA SCBs for M365 help secure federal information assets stored within M365 cloud business application environments through consistent, effective, and manageable security configurations. CISA created baselines tailored to the federal government's threats and risk tolerance with the knowledge that every organization has different threat models and risk tolerance. Non-governmental organizations may also find value in applying these baselines to reduce risks.
+
+The information in this document is provided "as is" for INFORMATIONAL PURPOSES ONLY. CISA does not endorse any commercial product or service, including any subjects of analysis. Any reference to specific commercial entities or commercial products, processes, or services by service mark, trademark, manufacturer, or otherwise does not constitute or imply endorsement, recommendation, or favoritism by CISA. This document does not address, ensure compliance with, or supersede any law, regulation, or other authority. Entities are responsible for complying with any recordkeeping, privacy, and other laws that may apply to the use of technology. This document is not intended to, and does not, create any right or benefit for anyone against the United States, its departments, agencies, or entities, its officers, employees, or agents, or any other person.
+
+This document tracks policies that have been discontinued from the security configuration baselines.
```

Comment:
I'd recommend moving this line to the top of this section. First, as it is it's easy to miss; users used to seeing the same legal disclaimer (like me) might just skip to the next section, assuming they've already read everything that follows. Putting it at the top would highlight it. Second, that would make this section consistent with the other baselines. This section in the other baselines always leads with a description of the applicable product followed by background on SCuBA; you've switched it here.

Reply:
will move to the top!